src/features/ITGovernance/Admin/Dates/index.tsx

```diff
@@ -50,228 +50,315 @@ const Dates = ({
+          }
         }
+      });
+
+      const newGrtDate = DateTime.fromObject(
+        {
+          day: Number(grtDateDay),
+          month: Number(grtDateMonth),
+          year: Number(grtDateYear)
+        },
+        { zone: 'UTC' }
+      ).toISO();
+
+      let newGrbDate: string | null = null;
+      if (grbReviewType !== SystemIntakeGRBReviewType.ASYNC) {
+        newGrbDate = DateTime.fromObject(
+          {
+            day: Number(grbDateDay),
+            month: Number(grbDateMonth),
+            year: Number(grbDateYear)
+          },
+          { zone: 'UTC' }
+        ).toISO();
       }
-    }).then(() => {
+
+      await updateReviewDates({
+        variables: {
+          input: {
+            id: systemId,
+            grtDate: newGrtDate,
+            ...(newGrbDate ? { grbDate: newGrbDate } : {})
+          }
+        }
+      });
+
       history.push(`/it-governance/${systemId}/intake-request`);
-    });
+    } catch (error) {
+      showMessage(t('error.message'), { type: 'error' });
+    }
   };
 
   return (
-    <Formik
-      initialValues={initialValues}
-      onSubmit={onSubmit}
-      validationSchema={DateValidationSchema}
-      validateOnBlur={false}
-      validateOnChange={false}
-      validateOnMount={false}
-    >
-      {(formikProps: FormikProps<SubmitDatesForm>) => {
-        const { errors, setErrors, handleSubmit } = formikProps;
-        const flatErrors = flattenErrors(errors);
-        return (
-          <>
-            {Object.keys(errors).length > 0 && (
-              <ErrorAlert
-                testId="system-intake-errors"
-                classNames="margin-top-3"
-                heading={t('form:inputError.checkFix')}
-              >
-                {Object.keys(flatErrors).map(key => {
-                  return (
-                    <ErrorAlertMessage
-                      key={`Error.${key}`}
-                      errorKey={key}
-                      message={flatErrors[key]}
+    <>
+      <PageHeading data-testid="grt-dates-view" className="margin-top-0">
+        {t('governanceReviewTeam:dates.heading')}
+      </PageHeading>
+      <h2>{t('governanceReviewTeam:dates.subheading')}</h2>
+      <Form onSubmit={handleSubmit(onSubmit)} className="maxw-none grid-col-8">
+        <FieldGroup
+          error={
+            !!errors.grtDateDay || !!errors.grtDateMonth || !!errors.grtDateYear
+          }
+        >
+          <fieldset className="usa-fieldset margin-top-4">
+            <legend className="usa-label margin-bottom-1">
+              {t('governanceReviewTeam:dates.grtDate.label')}
+            </legend>
+            <HelpText id="TestDate-DateHelp-grt">
+              {t('governanceReviewTeam:dates.grtDate.format')}
+            </HelpText>
+            <FieldErrorMsg>{errors.grtDateMonth?.message}</FieldErrorMsg>
+            <FieldErrorMsg>{errors.grtDateDay?.message}</FieldErrorMsg>
+            <FieldErrorMsg>{errors.grtDateYear?.message}</FieldErrorMsg>
```

Comment:
This works as-is, but just a heads up that React Hook Forms has its own `ErrorMessage` component. It automatically displays the error message, and scrolls to the error on submit.

```tsx
<ErrorMessage
  errors={errors}
  name="grtDateYear"
  as={FieldErrorMsg}
/>
```